components/organisms/SubdomainsTabBody.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, Modal, Skeleton } from "@ensdomains/thorin";
+import { Table } from "../02-molecules";
+import { DomainData } from "@/lib/domain-page";
+import { useState } from "react";
+import { CreateSubdomainModalContent } from "./CreateSubdomainModalContent";
+import { useRouter } from "next/router";
+
+export interface SubdomainsTabProps {
+  domainData: DomainData | null;
+  fetchDomainData: () => void;
+}
+
+export const SubdomainsTabBody = ({
+  domainData,
+  fetchDomainData,
+}: SubdomainsTabProps) => {
+  const [createSubdomainModalOpen, setCreateSubdomainModalOpen] =
+    useState(false);
+
+  const subdomainsArray: string[] | undefined = domainData?.subdomains.map(
+    (domain) => domain?.name as string
+  );
+
+  const router = useRouter();
+  const { name } = router.query; // Dynamic route parameter
```

Comment:
Can these components receive the searchedName instead of get it from the URL? As explained in another comment, it makes then not attached to a specific route

Reply:
@FrancoAguzzi  What do you mean?

Reply:
searchedName is defined by the URL we're accessing